docs/reference/components/sources/postgresql_metrics.cue

```diff
@@ -81,7 +81,8 @@ components: sources: postgresql_metrics: {
 		}
 		include_databases: {
 			description: """
-				A list of databases to match against the `datname` column for which you want to collect metrics from.
+				A list of databases to match (by using [POSIX Regular Expressions][urls.postgresql_matching]) against
```

Comment:
Same here, I might add an example using a regex to the example list. We may want to make it clearer that any plain string used for these will be used as a "subset" match (e.g. using `include_databases: ["foo"]` will automatically include `foobar`). Maybe we could add examples using anchors to show how to restrict this? Like `["^foo$"]`?